public/wp-content/themes/ColoredCow/landing.php

```diff
@@ -8,12 +8,23 @@
 
    	$currentdate=date('Y-m-d');
    	$posts = get_posts(array(
```

Comment:
try to do this with a query. Read about how to query a post in WP. Also Prefer to use functions provided in acf documentation.